utils/config/examples/daos_server_local.yml

```diff
@@ -0,0 +1,37 @@
+# access_points: ['localhost:10001']
```

Comment:
including this line is possibly confusing at this point in time

Reply:
It is commented and was required for me to test pool creation via daos_shell.